### ISSUE TITLE
Feature Evaluation

### ISSUE THREAD

**Issue by avloss** [catboost/catboost#429]
During this talk - https://youtu.be/8o0e-r0B5xQ?t=26m3s Anna mentions "New Feature Evaluation" functionality - I wasn't able to find anything in the docs, found some references in code
https://github.com/catboost/catboost/blob/master/catboost/python-package/catboost/eval/catboost_evaluation.py

Is this the functionality she's referring to? The documentation is insufficient (or maybe it's just me..), why does one need to pass in `path_to_dataset` and not the dataset itself?

BTW. this is an absolutely amazing idea - I was banging my head against the wall trying to find a reliable way to evaluate new features, as Anna said - plugging them into the model and comparing results is just not reliable/fast enough. So once  `catboost_evaluation` is fully documented and working it would be an amazing USP for your great library! :)

**Comment by avloss** [comment]
Thank for response - the tutorial includes

```
pool_dir = "amazon/"
path_to_dataset = pool_dir + 'train_all3'  # Path to the dataset.
description_file = pool_dir + 'train_full3.cd' # Path to the description file of the dataset.
```

It's not obvious to me how this dataset should be created. In other examples

```
from catboost import datasets
df, _ = = datasets.amazon()
```

is used, which makes perfect sense. It would be nicer (and more consistent) if feature evaluation algo would also accept DataFrame (instead of a file path).

**Comment by annaveronika** [comment]
You're right, this tutorial was done before we added catboost.datasets. We'll change it in all tuturials.

**Comment by kmedved** [comment]
Just came across this, and wanted to add my excitement for seeing documentation of this feature. From the example, it looks very useful.

**Comment by fantauzzi** [comment]
As per today, cannot find CatboostEvaluation in the documentation; is it supported?
Also, the tutorial is quite obscure.
